docs/doc-requirements.txt

```diff
@@ -1,3 +1,4 @@
 Sphinx==5.1.1
 sphinx-ansible-theme==0.9.1
+sphinx-immaterial==0.9.1
 sphinxcontrib-apidoc==0.3.0
```

Comment:
Since this PR replaces the use of this extension, it no longer has to be installed.
```suggestion
```